src/test/java/com/marizueva/laboratory/hw2/ex2/LoginTestWithSoftAssertions.java

```diff
@@ -0,0 +1,265 @@
+package com.marizueva.laboratory.hw2.ex2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class LoginTestWithSoftAssertions {
+    SoftAssert softAssert;
+
+    private WebDriver driver;
+    private  final String baseUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+    private final String login = "Roman";
+    private final String password = "Jdi1234";
+
+    @BeforeClass
+    public void beforeClass() {
+        softAssert = new SoftAssert();
+
+        WebDriverManager.chromedriver().setup();
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--disable-notifications", "--disable-popup-blocking");
+
+        driver = new ChromeDriver(options);
+
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+
+    }
+
+    @Test
+    public void testLoginTest() {
+        //Open test site by URL https://jdi-testing.github.io/jdi-light/index.html
+        // Test site is opened
+        // Assert Browser title"Home Page"
+        driver.navigate().to(baseUrl);
+        softAssert.assertEquals(driver.getTitle(), "HomePage");
+
+        //Perform login username: Roman pass: Jdi1234
+        WebElement openLoginDropdownButton = driver.findElement(By
+                .xpath("//li[@class='dropdown uui-profile-menu']"
+                        + "//span[@class='caret'][1]"));
+        openLoginDropdownButton.click();
+
+        WebElement loginField = driver.findElement(By
+                .xpath("//ul/li[@class='dropdown uui-profile-menu open']"
+                        + "/descendant::input[@id='name']"));
+        loginField.sendKeys(login);
+
+        WebElement passwordField = driver.findElement(By
+                .xpath("//ul/li[@class='dropdown uui-profile-menu open']"
+                        + "/descendant::input[@id='password']"));
+        passwordField.sendKeys(password);
```

Comment:
Why do you do not use `By.id("locator")`?

Reply:
done

Reply:
done